frontend/src/routes/(voters)/questions/+page.svelte

```diff
@@ -5,13 +5,17 @@
   const firstQuestionUrl = `${$page.url.pathname.replace(/\/$/, '')}/${$page.data.questions[0].id}`;
 </script>
 
-<div class="flex flex-col items-center gap-4 p-4">
-  <h1 class="text-3xl font-medium leading-6 text-gray-500">
-    {$page.data.appLabels.viewTexts.yourOpinionsTitle}
-  </h1>
-  <div class="my-2">
-    <p>{$page.data.appLabels.viewTexts.yourOpinionsDescription}</p>
-  </div>
+<section class="flex max-w-xl flex-grow flex-col justify-center p-lg">
+  <hgroup>
+    <h1>
+      {$page.data.appLabels.viewTexts.yourOpinionsTitle}
+    </h1>
+  </hgroup>
+  <p class="text-center">{$page.data.appLabels.viewTexts.yourOpinionsDescription}</p>
   <a href={firstQuestionUrl} class="btn-primary btn"
     >{$page.data.appLabels.actionLabels.startQuestions}</a>
-</div>
+</section>
+<section class="p-lg text-center text-sm text-secondary">
```

Comment:
should this be only a `div`? https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section

Reply:
True, these are better as `div`s here.